src/lightning_app/CHANGELOG.md

```diff
@@ -46,7 +46,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
--
+
+- Fixed an inconsistency where the flow won't be returning all its children flows ([#15466](https://github.com/Lightning-AI/lightning/pull/15466))
```

Comment:
that was a design choice from the beginning, not a bug. `.flows` returned the children